sqldelight-runtime/build.gradle

```diff
@@ -14,10 +14,6 @@ android {
     textOutput 'stdout'
   }
 
-  dexOptions {
-    preDexLibraries = !rootProject.ext.isCi
```

Comment:
Was that the only use of this flag?

Reply:
Sample module had it too

Reply:
I mean the `ci` flag in the root. Can we kill that or is it used elsewhere?

Reply:
Let me check

Reply:
Removed it